ICON-CREATION.md

```diff
@@ -2,63 +2,40 @@
 
 Skin uses the SVG tag to deliver all iconography. This guide will help you properly add or modify SVG files for the Skin library.
 
-## Step 1: Set up the icon
+## Step 1: Import the SVG
 
-First you should use a graphics editor, like Illustrator or Sketch, to export the SVG markup. For our examples we will use Sketch.
+In order to import the SVG into Skin, all you need to do is drop the SVG into `src/svg/icon` directory. The icon name will be the way the icon will be imported into Skin. If the icon does not have `icon-` prefix, it will be renamed automatically to include the prefix.
+The viewbox given will be translated to the `width` and `height` that the icons is going to use.
 
--   Open the necessary icon Sketch file
--   Copy the icon layer (right-click > copy ... or [Command ⌘] + [c] on Mac when the layer is highlighted). This should be the 24x24 layer only. It should contain the glyph altogether, with the 24x24 bounding box and the paths.
```

Comment:
We might want to retain some brief knowledge/understanding around art boarded vs cropped. I.e. by default all of our icons should have an artboard, but in some rare/bespoke cases cropped versions might be required. We can link out to the CSS tricks article on cropped vs artboard maybe.

Reply:
Okay, let's save it for a later update/pr